src/main/java/org/prebid/server/bidder/huaweiads/model/request/AdSlot30.java

```diff
@@ -0,0 +1,40 @@
+package org.prebid.server.bidder.huaweiads.model.request;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+import lombok.Value;
+
+import java.util.List;
+
+@Value
+@Builder(toBuilder = true)
+public class AdSlot30 {
+
+    @JsonProperty("slotid")
+    String slotId;
+
+    @JsonProperty("adtype")
+    Integer adType;
+
+    @JsonProperty("test")
```

Comment:
Redundant annotation. Jackson will map lowercase field names 1 to 1 to json field names. 
Please, remove and check for similar occurences.

Reply:
It's just a robust and safe way to avoid unnecessary mistakes in case of fields renaming. Other than that, why do we need to rely on some internal Jackson mapping logic, that also might be changed with the version update?